capture.py

```diff
@@ -160,12 +165,31 @@ def update_savings(self, savings):
         reply = urequests.get(url)
         return reply
 
+    def read_coin(self):
+        self.led.on()
+        for i in range (num_measures):
+            reply, (x, y, z, t), ack = read_sensor(self.i2c)
+            x_array[i] = x
+            y_array[i] = y
+            z_array[i] = z
+        self.led.off()
+    
+    def save_readings(self):
+        with open('readings.csv', 'w') as f:
+            for i in range (num_measures):
+                line = '{x},{y},{z}\n'.format(
```

Comment:
Maybe we could add column names before the loop?

In example: `f.write('x,y,z\n')`